default.nix

```diff
@@ -100,12 +123,13 @@ in
       default = {};
     };
 
-    vmailUIDStart = mkOption {
+    vmailUID = mkOption {
```

Comment:
We should keep this rename no matter if we merge the PR or not. I kind of used that as both the vmail uid and the start of the uid's.

Reply:
Makes sense to me!